API/NoDocs/Microsoft.Extensions.DependencyInjection/RedactionServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,12 @@
+// Assembly 'Microsoft.Extensions.Compliance.Redaction'
+
+using System;
+using Microsoft.Extensions.Compliance.Redaction;
+
+namespace Microsoft.Extensions.DependencyInjection;
+
+public static class RedactionServiceCollectionExtensions
+{
+    public static IServiceCollection AddRedaction(this IServiceCollection services);
+    public static IServiceCollection AddRedaction(this IServiceCollection services, Action<IRedactionBuilder> configure);
+}
```

Comment:
Is it missing `public static IServiceCollection AddRedaction(this IServiceCollection services, IConfigurationSection section);`

Reply:
This configuration state cannot be represented in JSON at the moment.